media/css/pebbles/components/_buttons-download.scss

```diff
@@ -172,20 +162,4 @@ ul.download-list {
     }
 }
 
-// Android architecture detection
```

Comment:
As far as I can tell all Android download buttons now point to the Play Store, so these selectors aren't used anywhere.